bot/exts/info/tags.py

```diff
@@ -24,145 +29,217 @@
 FOOTER_TEXT = f"To show a tag, type {constants.Bot.prefix}tags <tagname>."
 
 
+class COOLDOWN(enum.Enum):
+    """Sentinel value to signal that a tag is on cooldown."""
+
+    obj = object()
+
+
+class TagIdentifier(NamedTuple):
+    """Stores the group and name used as an identifier for a tag."""
+
+    group: Optional[str]
+    name: str
+
+    def get_fuzzy_score(self, fuzz_tag_identifier: TagIdentifier) -> float:
```

Comment:
What's wrong with simply calling that `identifier` or `match`?

Reply:
Nothing wrong, but I wanted to make it clear that the method is fuzzy matching the passed in identifier with self as the target